vcpkg/maintainers/functions/vcpkg_make_configure.md

```diff
@@ -0,0 +1,159 @@
+---
+title: vcpkg_make_configure
+description: Use vcpkg_make_configure to configure a Make-based project.
+ms.date: 01/30/2024
+---
+# vcpkg_make_configure
+
+Configure a Make-based project.
+
+## Usage
+        
+```cmake
+vcpkg_make_configure(
+    [AUTOCONFIG]
+    [COPY_SOURCE]
+    [DISABLE_MSVC_WRAPPERS]
+    [NO_CPPFLAGS]
+    [ADD_BIN_TO_PATH]
+    [NO_DEFAULT_OPTIONS]
+    [NO_MSVC_FLAG_ESCAPING]
+    [USE_RESPONSE_FILES]
+    [SOURCE_PATH <source-path>]
+    [SHELL <shell>]
+    [OPTIONS <configure-setting>...]
+    [OPTIONS_RELEASE <configure-setting>...]
+    [OPTIONS_DEBUG <configure-setting>...]
+    [PRE_CONFIGURE_CMAKE_COMMANDS <cmake-command>...]
+    [POST_CONFIGURE_CMAKE_COMMANDS <cmake-command>...]
+    [LANGUAGES <language>...]
+)
+```
+
+To use this function, you must depend on the helper port `vcpkg-make`:
+
+```json
+"dependencies": [
+  {
+    "name": "vcpkg-make",
+    "host": true
+  }
+]
+```
+
+## Parameters
+
+### AUTOCONFIG
+
+Boolean value that runs autoreconf when set.
+
+### COPY_SOURCE
+
+If specified, the source directory will be copied to the build directory before the configuration process begins. Useful for projects that modify the source tree during configuration or build.
+
+### DISABLE_MSVC_WRAPPERS
+
+Disables the use of MSVC compiler wrappers. Useful for projects that are not compatible with the wrappers.
```

Comment:
Should probably contain more information what those wrappers are actually doing? In general these are automake wrappers `ar-lib` (covnersion ar flags to lib flags) and `compile` (conversion and reordering of flags) but also custom vcpkg wrappers like `windres-rc` (take windres args -> convert to rc) , `cl-cpp-wrapper` (allow processing from stdin)